packages/components/app/styles/components/table.scss

```diff
@@ -36,8 +41,27 @@
   padding: 20px 16px;
 }
 
+
 .hds-table__thead {
-  border-bottom: 1px solid var(--token-color-border-primary);
+  .hds-table__tr {
+    color: #0c0c0e;
+    background-color: #f1f2f3;
+
+    &:first-of-type {
+      border-top-left-radius: $hds-table-inner-border-radius;
+      border-top-right-radius: $hds-table-inner-border-radius;
```

Comment:
I _think_ that if you move the color/background declarations there are above (for the row) inside the `hds-table__th, .hds-table__th-sort {}` selector, probably you can remove these two (there's no background color, no need to set a border radius)